### PR TITLE
Add note for CentOS7 repo dependencies

### PR DESCRIPTION
**Pull request by hunter86bg**

### CODE REVIEW

File: docs/Developer-guide/Building-GlusterFS.md [hunk 1]
```diff
@@ -249,8 +249,7 @@ steps to build RPMs,
 
 This will create rpms from the source in 'extras/LinuxRPM'. *(Note: You
 will need to install the rpmbuild requirements including rpmbuild and
-mock)
-
+mock)<br>
```
Comment (kshithijiyer) [inline docs/Developer-guide/Building-GlusterFS.md:252]: I don't see the need for removing * and adding <br>, if you still wish to do so please remove * before `(Note:` as well

File: docs/Developer-guide/Building-GlusterFS.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ The following yum command installs the build requirements for CentOS / Enterpris
     pyxattr readline-devel rpm-build sqlite-devel systemtap-sdt-devel \
     tar userspace-rcu-devel
 ```
-Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package
+Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package<br>
```
Comment (kshithijiyer) [inline docs/Developer-guide/Building-GlusterFS.md:70]: This should be
```
**Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package**
```
Which will give the below output:
**Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package**

**Comment by hunter86bg** [comment]
OK

File: docs/Developer-guide/Building-GlusterFS.md (hunk 2 above)
Comment (hunter86bg) [reply docs/Developer-guide/Building-GlusterFS.md:70]: I have left the opportunity any reviewer to modify the code.
Anyway - formatting for me is not so important than the info that the user needs the CentOS SIG repos for some packages.

File: docs/Developer-guide/Building-GlusterFS.md (hunk 1 above)
Comment (amarts) [reply docs/Developer-guide/Building-GlusterFS.md:252]: https://github.com/gluster/glusterdocs/blob/35d17d7355a6223aec308e0d928f5d8f488d3288/docs/Developer-guide/Building-GlusterFS.md 

The above comment is valid. Adding <br> at end can happen with a '*' too. But without that, the highlighting goes for a toss!

**Comment by hunter86bg** [comment]
Now it should look better

**Pull request merged**